### PR TITLE
index-page event handler

### PR DESCRIPTION
**Pull request by theethaj**
I already complete this event handler. Please review it, thank you.

### CODE REVIEW

File: quizer_game/templates/quizer_game/index.html [hunk 1]
```diff
@@ -1,10 +1,115 @@
...
+      <div class="top">
+        <a class="create" href="">Create quiz</a>
+        <a class="history" href="">History</a>
+        <a class="logout" href="">Logout</a>
+      </div> -->
+
+      <div class="top">
+        <a href="/quizer/login/" class="signup">Sign Up</a>
+        <a href="/quizer/login/" class="login">Login</a>
+      </div>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/index.html]: Shouldn't use hard code url. Please consider fixing it.

File: quizer_game/templates/quizer_game/index.html [hunk 2]
```diff
@@ -1,10 +1,115 @@
...
+          <div class="row slider-text align-items-center justify-content-center">
+            <div class="col-md-10 ftco-animate text-center sizey">
+              <h1 class="mb-4" style="color: black;">Welcome to
+                <strong class="typewrite" data-period="4000" data-type='[ "Quizer.", "Quizer." ]'>
+                  <span class="wrap"></span>
+                </strong>
+              </h1>
+              <p1><a href="/quizer/select-player-name/" class="btn btny px-4 py-3"><span class="ion-ios-play mr-2"></span> Let's play</a></p1>
+              <p1><a href="#how_to_play" class="btn btny js-scroll-trigger px-4 py-3"><span class="ion-ios-play mr-2"></span> How to play</a></p1>
+              <p1><a href="/quizer/leaderboard-index/" class="btn btny px-4 py-3"><span class="ion-ios-play mr-2"></span> Leaderboard</a></p1>
```
Comment (NutthanichN) [inline quizer_game/templates/quizer_game/index.html]: Shouldn't use hard code url. Please consider fixing it.

File: quizer_game/templates/quizer_game/index.html (hunk 1 above)
Comment (theethaj) [reply quizer_game/templates/quizer_game/index.html]: I've already improve it. Please review again.

File: quizer_game/templates/quizer_game/index.html (hunk 2 above)
Comment (theethaj) [reply quizer_game/templates/quizer_game/index.html]: I've already improve it. Please review again.

**Comment by Wijantra** [comment]
We require 2 spaces for HTML.
but some space in index.html have not 2 spaces

**Comment by Wijantra** [comment]
> 
> 
> We require 2 spaces for HTML.
> but some space in index.html have not 2 spaces
> please check it

Sorry I'm wrong,  index.html have 2 spaces

**Pull request merged**